src/aerialmap_display.cpp

```diff
@@ -96,11 +100,11 @@ AerialMapDisplay::AerialMapDisplay() : Display(), dirty_(false), received_msg_(f
   blocks_property_->setMax(maxBlocks);
 
   frame_convention_property_ =
-      new EnumProperty("Frame Convention", "XYZ -> ENU", "Convention for mapping cartesian frame to the compass", this,
-                       SLOT(updateFrameConvention()));
+      new EnumProperty("Frame Convention", "XYZ -> ENU",
```

Comment:
I don't see the value of having a `Frame Convention` property with only one option.

Reply:
That's a hint to people previously using other frame conventions, and I prefer it over simply deleting this option without telling which one is used now.